src/session.rs

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2017, 2022 ZettaScale Technology Inc.
+// Copyright (c) 2024 ZettaScale Technology
```

Comment:
I think we don't need to update the year here.

Reply:
You've a put a 👍 on my [comment](https://github.com/eclipse-zenoh/zenoh-python/pull/184#issuecomment-2068809841), so I think the thread can be resolved.